Nightly-Tests/Sync-Tests/createCluster.sh

```diff
@@ -79,6 +79,8 @@ fi
 
 echo "Tearing down cluster" 
 timeout 10 tiab/teardown_cluster.py -t $UUID -n $K8S_NAMESPACE
+kubectl delete pods --all
+kubectl delete services --all
```

Comment:
Why these two lines? The teardown script should take care of all resources created with the UUID tag: pods, services and config maps.

Reply:
Aaaah remnant from local changes. `teardown_cluster.py` doesn't always tear down the clusters when i run it locally. I'll pull those out.